app/api/src/main/java/com/thechance/api/endpoints/ProductsRoutes.kt

```diff
@@ -27,25 +34,35 @@ fun Route.productsRoutes() {
         authenticate(API_KEY_AUTHENTICATION) {
             get("/{productId}/categories") {
                 val productId = call.parameters["productId"]?.trim()?.toLongOrNull()
-                val categories = productUseCasesContainer.getCategoriesForProductUseCase(productId = productId)
-                    .map { it.toApiCategoryModel() }
+                val categories =
+                    productUseCasesContainer.getCategoriesForProductUseCase(productId = productId)
+                        .map { it.toApiCategoryModel() }
                 call.respond(ServerResponse.success(categories))
             }
 
             get("/{productId}") {
                 val productId = call.parameters["productId"]?.trim()?.toLongOrNull()
-                val categories = productUseCasesContainer.getProductDetailsUseCase(productId = productId)
-                    .toApiProductModel()
+                val categories =
+                    productUseCasesContainer.getProductDetailsUseCase(productId = productId)
+                        .toApiProductModel()
                 call.respond(ServerResponse.success(categories))
             }
 
+            get("/recent-products") {
+                val products =
```

Comment:
Please change the route to 'recentProducts' or 'recent', since it is under the 'product' route. For example, the URL would be '/product/recent'.